docs/markdown/Style-guide.md

```diff
@@ -82,6 +86,6 @@ sources = files([
   'f1.c',
   'f2.c',
   'f10.c',
-  'f20.c'
+  'f20.c',
 ])
```

Comment:
```suggestion
  'f20.c',
)
```
In https://github.com/NixOS/nix/pull/10378#discussion_r1558205104 @eli-schwartz says he would prefer no array here